database/migrations/2015_11_23_092743_add_columns_to_users_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddColumnsToUsersTable extends Migration
```

Comment:
It expected space after function keyword(anonymous function) in this file.

*Oladipupo Isola | Software Developer, Andela*
*See what TechCrunch
<http://techcrunch.com/2015/06/25/backed-by-spark-capital-andela-will-develop-a-continent-of-tech-talent/>
and StackOverflow
<http://blog.stackoverflow.com/2015/05/stack-overflow-and-andela-partner-to-provide-education-beyond-borders/>
say
about us!*
(c) +234-703-302-0216 | Andela.com <http://andela.com/>

On Tue, Nov 24, 2015 at 6:56 AM, Prosper Otemuyiwa <notifications@github.com
> wrote:

> In database/migrations/2015_11_23_092743_add_columns_to_users_table.php
> <https://github.com/andela/pibbble/pull/148#discussion_r45699286>:
>
> > @@ -0,0 +1,37 @@
> > +<?php
> > +
> > +use Illuminate\Database\Schema\Blueprint;
> > +use Illuminate\Database\Migrations\Migration;
> > +
> > +class AddColumnsToUsersTable extends Migration
>
> But this is a migration file not tests. Ignore that
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela/pibbble/pull/148/files#r45699286>.
>